velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -232,6 +232,10 @@ class S3FileSystem::Impl {
   Impl(const S3Config& s3Config) {
     VELOX_CHECK(getAwsInstance()->isInitialized(), "S3 is not initialized");
     Aws::S3::S3ClientConfiguration clientConfig;
+    clientConfig.checksumConfig.requestChecksumCalculation =
```

Comment:
Can you add a comment here that these are required for Oracle Cloud? Is there any Oracle Cloud documentation regarding this?